### DISCUSSION TITLE
Discovery Home assistant

### DISCUSSION THREAD

**Discussion by meijer3** [MichielVanwelsenaere/HomeAutomation.CoDeSys3#111] [Ideas]
Hi,

First of all, great work!
Is possible to use the dicovery function of MQTT/HASS. 
So your code could auto generate all the switches etc on the fly.
https://www.home-assistant.io/docs/mqtt/discovery/

**Comment by MichielVanwelsenaere** [comment]
Hi,

thanks, much appreciated!
MQTT discovery functionality is currently not included in the project.
Reasons/thoughts etc have already been discussed in #77 

I'm not going to say 'never' but for me the priorities are currently more on the RS485 and HVAC topics. 
Who knows, perhaps after that but that's not for the first 1-2 years.
